airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,401 @@
+
+        try:
+            self.hook.delete_table(
+                self.project_id,
+                self.instance_id,
+                self.table_id,
+            )
+        except google.api_core.exceptions.NotFound:
+            # It's OK if table doesn't exists
+            self.log.info("The Table '%s' does not exists already.", self.table_id)
+        except google.api_core.exceptions.GoogleAPICallError as e:
+            self.log.error('An error occurred. Exiting.')
+            raise e
+
+
+class BigTableClusterUpdateOperator(BaseOperator, BigTableValidationMixin):
+    """
+    Updates a BigTable cluster.
+
+    For more details about updating cluster have a look at the reference:
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/cluster.html#google.cloud.bigtable.cluster.Cluster.update # noqa: E501
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the Instance the Table belongs
+    :type cluster_id: str
+    :param cluster_id: The ID of the Cluster to update
+    :type nodes: int
+    :param nodes: Desired number of nodes for the Cluster
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'cluster_id', 'nodes')
+    template_fields = ['project_id', 'instance_id', 'cluster_id', 'nodes']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 cluster_id,
+                 nodes,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.cluster_id = cluster_id
+        self.nodes = nodes
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        super(BigTableClusterUpdateOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        instance = self.hook.get_instance(self.project_id, self.instance_id)
+        if not instance:
+            raise AirflowException("Instance '{}' does not exists.".format(self.instance_id))
+
+        try:
+            self.hook.update_cluster(
+                instance,
+                self.cluster_id,
+                self.nodes
+            )
+        except google.api_core.exceptions.NotFound:
+            raise AirflowException("Cluster '{}' does not exists for Instance '{}'.".format(
+                self.cluster_id,
+                self.instance_id
+            ))
+        except google.api_core.exceptions.GoogleAPICallError as e:
+            self.log.error('An error occurred. Exiting.')
+            raise e
+
+
+class BigTableTableWaitForReplicationSensor(BaseSensorOperator, BigTableValidationMixin):
+    """
+    Sensor that waits for BigTable Table to be fully replicated on its Clusters
+
+    For more details about cluster states for a table have a look at the reference:
```

Comment:
nit: Sometimes you write `Cluster`/`cluster`, `Table`/`table` etc.
Maybe would be good to unify the usages?